samples/code.hera

```diff
@@ -0,0 +1,9 @@
+```
+const that = this;
```

Comment:
Maybe replace this with `export foo = 42`, and check for its existence in the test?